execution_engine/src/core/runtime/auction_internal.rs

```diff
@@ -76,9 +76,9 @@ where
             .map_err(|exec_error| <Option<Error>>::from(exec_error).unwrap_or(Error::Storage))
     }
 
-    fn read_withdraw(&mut self, account_hash: &AccountHash) -> Result<Vec<UnbondingPurse>, Error> {
-        match self.context.read_gs(&Key::Withdraw(*account_hash)) {
-            Ok(Some(StoredValue::Withdraw(unbonding_purses))) => Ok(unbonding_purses),
+    fn read_unbond(&mut self, account_hash: &AccountHash) -> Result<Vec<UnbondingPurse>, Error> {
```

Comment:
I'm now wondering whether it won't become a problem, similar to auction state, where after an upgrade new node won't be able to load auction state (in this case unbonding purses) from before it.

Reply:
Discussed offline